torchbox/torchbox.com
# Service page - featured case study block - BE

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1371196780)

### Description of Changes Made

This PR updates the `ServiceStoryBlock` so that it now includes a `FeaturedCaseStudyBlock`. It picks up from the work done in #60. Additional, the template and pattern library is updated to ensure everything works as before. The stylesheet is also updated to ensure that the featured image and logo are displayed -- Kudos to @chris-lawton for this (See e20aaad)

The Featured case study block allows content editors to

- add a tagline
- specify a work page or historical work page, which is used as the basis for both the featured case study image and logo (this can be overriden by the editor, as indicated in the next  item below)
- specify optional image and logo to override what the specified page above provides
- add optional rich text

> **Note**
>
> The Work Page is supposed to have a `logo` field. This is already being handled in #72, so here we just have a placeholder to prevent breaking the functionality:

https://github.com/torchbox/torchbox.com/blob/f429a8f0d93e5b3af0f51c8d73d58bf6248b9660/tbx/work/models.py#L167

### How to Test

- Create a Service Page
- In the body StreamField, you should be able to add a Featured Case Study Block (with different combinations of link, tagline, text, image, logo), and it should render as expected on the template

### Screenshots

<details>
  <summary>Expand to see more</summary>

![image](https://github.com/torchbox/torchbox.com/assets/7713776/b967578d-93c0-42f2-9bfa-8132bb879f26)

![image](https://github.com/torchbox/torchbox.com/assets/7713776/abb878e7-b20d-4090-a841-53c3dd077784)

![image](https://github.com/torchbox/torchbox.com/assets/7713776/4c4c852a-9c71-4b3f-89b8-5267cf17ae21)

![image](https://github.com/torchbox/torchbox.com/assets/7713776/ea22f045-46b0-40ee-8687-83f63f7db33b)

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on Linux
  - Latest version of Firefox on Linux

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
